src/schemathesis/schemas.py

```diff
@@ -105,11 +109,35 @@ def get_all_endpoints(self) -> Generator[Endpoint, None, None]:
                 continue
             common_parameters = get_common_parameters(methods)
             for method, definition in methods.items():
-                if method == "parameters" or should_skip_method(method, self.method):
+                print(definition)
+                if (
+                    method == "parameters"
+                    or should_skip_method(method, self.method)
+                    or (definition and should_skip_endpoint(definition.get("tag"), self.tag))
```

Comment:
It seems like the first part of the condition (`definition`) will become irrelevant after fixing the schema in the failing test

Reply:
Also, it should be `tags` instead of `tag`